test/Yesod/Auth/SimpleSpec.hs

```diff
@@ -127,3 +127,11 @@ spec = withApp $ do
         statusIs 303
         ur <- runHandler getUrlRender
         followRedirect >>= assertEq "redirection successful" (Right $ ur HomeR)
+
+    describe "with a hash generated from scrypt-0.5.0" $
+
+      it "should match one generated with password" $ do
```

Comment:
I think you should push this to master, or add it in a separate PR, and then rebase this branch of that.
To confirm it indeed didn't change :sweat_smile: 

Also the label should be something like "golden test to ensure the hashing is the same regardless of script implementation"